### PR TITLE
#712 Add crossview dialog checkbox for persisting manually

### PR DESCRIPTION
**Pull request by adrianchivu**
### Prerequisites

- [X] I have written a descriptive pull-request title
- [X] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/CDP4-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the CDP4-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/CDP4-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### CODE REVIEW

**Review by lxatrhea — APPROVED** [review]
I think we should get used to the fact that we should update headers to 2021.

**Review by cozminvelciu — CHANGES_REQUESTED** [review]
Will review after [parent PR](#684) is merged, as for now the diff is inconclusive. Being forced into sequential reviews and merges (instead of handling each issue separately) is one of the disadvantages of having multiple branches based on one another.

File: CDP4CrossViewEditor/ViewModels/WorkbookSelectionDialogResult.cs [hunk 1]
```diff
@@ -90,5 +95,10 @@ public class WorkbookSelectionDialogResult : BaseDialogResult
         /// Get or sets workbook manually edited data values
         /// </summary>
         public Dictionary<string, string> WorkbookChangedValues { get; private set; }
+
+        /// <summary>
+        ///
```
Comment (cozminvelciu) [inline CDP4CrossViewEditor/ViewModels/WorkbookSelectionDialogResult.cs:100]: Missing XML docs.

File: CDP4CrossViewEditor/Views/CrossViewDialog.xaml [hunk 2]
```diff
@@ -21,12 +21,19 @@
...
                 <RowDefinition Height="Auto" />
                 <RowDefinition Height="Auto" />
+                <RowDefinition Height="Auto" />
             </Grid.RowDefinitions>
             <Grid.ColumnDefinitions>
                 <ColumnDefinition Width="*" />
             </Grid.ColumnDefinitions>
-            <uc:ElementThingSelector Grid.Row="0" Mode="ElementBase" VerticalAlignment="Stretch" x:Name="ElementThingSelector" />
-            <uc:ParameterThingSelector Grid.Row="1" Mode="ParameterBase" VerticalAlignment="Stretch" x:Name="ParameterThingSelector" />
+            <dxlc:LayoutControl View="Group"
```
Comment (cozminvelciu) [inline CDP4CrossViewEditor/Views/CrossViewDialog.xaml:29]: One thing I suggest we do is move the checkbox and the text to the bottom of the dialog, and align it to the right, so it jumps out more when clicking OK / Cancel. Where it is now (upper left) makes it very easy to ignore/disregard.

**Review by cozminvelciu — CHANGES_REQUESTED** [review]

File: CDP4CrossViewEditor/Views/CrossViewDialog.xaml (hunk 2 above)
Comment (cozminvelciu) [reply CDP4CrossViewEditor/Views/CrossViewDialog.xaml:29]: I'm not saying this is the right way, just a suggestion that might be worth implementing _if_ you agree.

**Review by cozminvelciu — APPROVED** [review]

**Pull request merged**